client/src/styles/styles.scss

```diff
@@ -1,5 +1,10 @@
 @import 'colors.scss';
 
+@font-face {
+  font-family: 'Tommy';
+  src: local('Tommy'), url('../fonts/Tommy.ttf') format('truetype');
+}
+
 body {
   background-color: $light;
   color: $primary;
```

Comment:
Add `font-family: Tommy` to this body rule so that every page in our app uses this font. If you do this, you shouldn't need to add `font-family: Tommy` to any other CSS file/rule (which means you can take it out of the SessionSummary and DetailsModal CSS files)